src/DynamoCore/Models/CustomNodeWorkspaceModel.cs

```diff
@@ -196,20 +196,23 @@ protected virtual void OnFunctionIdChanged(Guid oldId)
 
         public override bool SaveAs(string newPath, ProtoCore.RuntimeCore runtimeCore, bool isBackUp = false)
         {
-            var originalPath = FileName;
-
-            // A SaveAs to an existing function id prompts the creation of a new 
-            // custom node with a new function id
-            if (originalPath != newPath)
+            if (!isBackUp)
```

Comment:
Yep we do, which is why I proposed to do this:

```csharp
if (isBackup)
    return base.SaveAs(newPath, runtimeCore);

if (originalPath != newPath)
{
    ...
}

return base.SaveAs(newPath, runtimeCore);
```

How does that look? Cleaner I personally think, but it's a minor thing really :)